federation/pkg/federation-controller/namespace/namespace_controller.go

```diff
@@ -74,13 +80,19 @@ type NamespaceController struct {
 
 // NewNamespaceController returns a new namespace controller
 func NewNamespaceController(client federation_release_1_4.Interface) *NamespaceController {
+
+	broadcaster := record.NewBroadcaster()
+	broadcaster.StartRecordingToSink(eventsink.NewFederatedEventSink(client))
+	recorder := broadcaster.NewRecorder(api.EventSource{Component: "namespace-controller"})
```

Comment:
Not sure where this is used and should this be unique across all controllers, but Component name "federation-namespace-controller" should be better?